beacon-chain/core/transition/transition_no_verify_sig.go

```diff
@@ -426,17 +423,36 @@ func electraOperations(
 	st state.BeaconState,
 	block interfaces.ReadOnlyBeaconBlock) (state.BeaconState, error) {
 	// 6110 validations are in VerifyOperationLengths
-
+	bb := block.Body()
+	bod, ok := bb.(interfaces.ROBlockBodyElectra)
+	if !ok {
+		return nil, errors.New("could not cast block body to electra block body")
+	}
 	// Electra extends the altair operations.
-	st, err := altairOperations(ctx, st, block)
```

Comment:
I can't use altairOperations because process deposits got changed in the middle of the function